elastic/kibana
# [Detection Engine] update data view select to include name and title

## Summary

Addresses https://github.com/elastic/kibana/issues/137823

- [ ] Confirmation from @ARWNightingale that this design suits

<img width="574" alt="Screenshot 2024-11-26 at 9 43 21 PM" src="https://github.com/user-attachments/assets/9099d2da-8086-4dbe-925f-67a48e407739">

<img width="577" alt="Screenshot 2024-11-26 at 9 43 27 PM" src="https://github.com/user-attachments/assets/6f4ad8a7-66d2-40af-9934-dfe65fad1e96">

### How to test

Go to rule creation and play around with data view selection! That's it. Make sure that it is rendered correctly, and that on save, it is saved as intended.

### Checklist
